run-xunit-perf.cmd

```diff
@@ -0,0 +1,56 @@
+@setlocal
+@echo off
+
+set HERE=%CD%
+set CORECLR_REPO=%CD%
+set CORECLR_OVERLAY=%CORECLR_REPO%\bin\tests\Windows_NT.x64.Release\Tests\Core_Root
+set CORECLR_PERF=%CORECLR_REPO%\bin\tests\Windows_NT.x64.Release\Jit\Performance\CodeQuality
+set RUNLOG=%HERE%\perfrun.log
```

Comment:
This seems to be the wrong location for a log file. Although it is not a build log, maybe a location would be the bin/log directory so that it can be cleaned by existing infrastructure.